.github/release.yml

```diff
@@ -0,0 +1,32 @@
+changelog:
+  categories:
+    - title: BREAKING CHANGES
+      labels:
+        - breaking
+    - title: NOTES
+      labels:
+        - note
+        - documentation
+    - title: FEATURES
+      labels:
+        - feature
+    - title: ENHANCEMENTS
+      labels:
+        - enhancement
+    - title: BUG FIXES
+      labels:
+        - bug
+    - title: INTERNAL
+      labels:
+        - dependencies
+        - ci
+        - "*"
+  exclude:
+    labels:
+      - duplicate
+      - wontfix
+      - question
+      - invalid
+      - needs-reproduction
+      - needs-research
+      - stale
```

Comment:
I think we don't actually have most of these - is it worth revising the list to reflect the reality?

Reply:
It does no harm to have them here, it's a catch if there is something that did get labeled as such since the labels exist. I can remove the labels from the repo and here if we think that's best, or can merge as is and we can revisit the labels in general.

Reply:
Removed nonexistant